docs/reference/mapping/fields/synthetic-source.asciidoc

```diff
@@ -61,10 +60,8 @@ types:
 ** <<numeric-synthetic-source,`long`>>
 ** <<range-synthetic-source,`range` types>>
 ** <<numeric-synthetic-source,`scaled_float`>>
-** <<search-as-you-type-synthetic-source,`search_as_you_type`>>
 ** <<numeric-synthetic-source,`short`>>
 ** <<text-synthetic-source,`text`>>
```

Comment:
`text` is controversial - if there is no keyword subfield it will store the text

Reply:
We should have a category "It's complicated" :P

Reply:
imo native means the field type handling synthetic source without relying on ignored source. It says nothing about how it implements synthetic source.